mmv1/third_party/terraform/services/iap/resource_iap_settings_test.go

```diff
@@ -0,0 +1,177 @@
+package iap_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccIapSettings_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":          envvar.GetTestOrgFromEnv(t),
+		"billing_account": envvar.GetTestBillingAccountFromEnv(t),
+		"random_suffix":   acctest.RandString(t, 10),
+		"role":            "roles/iap.settingsAdmin",
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
```

Comment:
`time` provider is needed for `time_sleep` resource. Example: https://github.com/hashicorp/terraform-provider-google/blob/main/google/services/iap/iam_iap_web_type_app_engine_generated_test.go#L102

Reply:
Okay